desktop/components/article/templates/mixins.jade

```diff
@@ -35,16 +35,12 @@ mixin fullscreen(article)
     .article-fullscreen-down-arrow
       a.icon-chevron-down
 
-    if article.get('is_super_article')
+    if article.get('is_super_article') || superArticle
```

Comment:
I think in this case it's safe to use `superArticle` because of [this line](https://github.com/artsy/force/blob/master/desktop/models/article.coffee#L62). Helps to clean up these lines a bit